.gitlab-ci.yml

```diff
@@ -0,0 +1,199 @@
+    paths:
+      - deployments
+
+test:prepare_acceptance:
+  image: docker
+  before_script: []
+  stage: test_prep
+  script:
+    - docker build -f Dockerfile.acceptance-testing -t $DOCKER_REPOSITORY:prtest .;
+    - docker save $DOCKER_REPOSITORY:prtest > tests_image.tar
+    - docker build -t testing -f tests/Dockerfile tests
+    - docker save testing > acceptance_testing_image.tar
+    - wget https://d1b0l86ne08fsf.cloudfront.net/mender-artifact/master/mender-artifact
+    - chmod +x mender-artifact
+  artifacts:
+    expire_in: 30min
+    paths:
+      - tests_image.tar
+      - acceptance_testing_image.tar
+      - mender-artifact
+
+test:acceptance_tests:
+  image: tiangolo/docker-with-compose
+  before_script: []
+  services:
+    - docker:dind
+  stage: test
+  dependencies:
+    - test:build_acceptance
+    - test:prepare_acceptance
+  script:
+    - apk add git bash
+    - 'export SHARED_PATH="$(dirname ${CI_PROJECT_DIR})"'
+    - git clone https://github.com/mendersoftware/integration.git ${SHARED_PATH}/mender-integration
+    - mkdir -p ${SHARED_PATH}/tests
+    - mv ${SHARED_PATH}/mender-integration/extra/travis-testing/* docs/* deployments mender-artifact tests/* ${SHARED_PATH}/tests
+    - docker load -i acceptance_testing_image.tar
+    - docker load -i tests_image.tar
+    - TESTS_DIR=${SHARED_PATH}/tests ${SHARED_PATH}/tests/run-test-environment acceptance ${SHARED_PATH}/mender-integration ${SHARED_PATH}/tests/docker-compose-acceptance.yml ;
+    - TESTS_DIR=${SHARED_PATH}/tests ${SHARED_PATH}/tests/run-test-environment acceptance ${SHARED_PATH}/mender-integration ${SHARED_PATH}/tests/docker-compose-acceptance.mt.yml ;
+    - mv ${SHARED_PATH}/tests/coverage-acceptance.txt $CI_PROJECT_DIR
+  artifacts:
+    expire_in: 30min
+    paths:
+      - coverage-acceptance.txt
+  tags:
+    - docker
+
+test:static:
+  stage: test
+  dependencies: []
+  services: []
+  script:
+    - curl -sL https://deb.nodesource.com/setup_11.x | bash -
+    - apt-get -qq update
+    - apt-get install -qy --allow-unauthenticated python-pip e2tools nodejs liblzma-dev
+    - pip install pyyaml
+    - npm install -g swagger-cli
+    # Get our own Swagger verifier
+    - wget https://raw.githubusercontent.com/mendersoftware/autodocs/master/verify_docs.py
+    # Test if code was formatted with 'go fmt'
+    # Command will format code and return modified files
+    # fail if any have been modified.
+    - if [ -n "$(go fmt)" ]; then echo 'Code is not formatted with "go fmt"'; false; fi
+    # Perform static code analysys
+    - go vet `go list ./... | grep -v vendor`
+    # Fail builds when the cyclomatic complexity reaches 15 or more
+    - gocyclo -over 15 `find . -iname '*.go' | grep -v 'vendor' | grep -v '_test.go'`
+    # Verify that the Swagger docs are valid
+    - swagger-cli validate docs/*.yml
+    # Verify that the Swagger docs follow the autodeployment requirements
+    - python2.7 verify_docs.py `find docs -name "*.yml"`
```

Comment:
We can change this to Python 3 call once we merge [this PR](https://github.com/mendersoftware/autodocs/pull/1).

Reply:
The mentioned PR is now merged, can we please switch to Python 3? Also in `device_auth`, `deviceadm`, `inventory`, `tenantadm`, `mender-cli` and `useradm`.